diesel/src/associations/belongs_to.rs

```diff
@@ -27,8 +27,9 @@ impl<Parent, Child, Iter> GroupedBy<Parent> for Iter where
         let id_indices: HashMap<_, _> = parents.iter().enumerate().map(|(i, u)| (u.id(), i)).collect();
```

Comment:
`parents` is a slice you immediately call `iter` on (two times)—why not take an `IntoIter<Item=Parent>` instead? Might be cheaper to clone/work with. Totally unimportant though, I guess you always deal with `Vec`s here anyway.